src/plugins/mmapstorage/internal.h

```diff
@@ -99,7 +124,12 @@ typedef struct _mmapFooter MmapFooter;
 /**
  * Mmap information header
  *
- * shall contain only fixed-width types
+ * Changes to this struct can/will break compatibility with existing files.
+ * If a breaking changed needs to be introduced, change `ELEKTRA_MAGIC_MMAP_NUMBER`
```

Comment:
Please give details what a breaking change is:

- reordering of struct?
- adding to struct?
- ...